pages/web-lit-basics.jsx

```diff
@@ -44,7 +44,7 @@ var curriculumList = [
         image1x: "/img/pages/web-lit-basics/img-kraken-code.jpg",
         image2x: "/img/pages/web-lit-basics/img-kraken-code@2x.jpg",
         subtitle: "Understanding credibility",
-        description: "Learners use the Internet to solve the mystery of The Kraken, a legendary sea creature, while also learning about search terms, keywords, and how to assess the validity and relevance of web sources.",
+        description: "Learners use the Internet to solve the mystery of The Kraken, a legendary sea creature, while also learning about search terms, keywords, and how to assess the validity and relevance of web sources. This activity was made by <a href='http://mouse.org'>MOUSE</a>.",
```

Comment:
@sabrinang @kristinashu for your ref, this is what we are talking about ^

<img width="765" alt="screen shot 2016-03-22 at 9 07 15 pm" src="https://cloud.githubusercontent.com/assets/2896608/13975397/19663362-f072-11e5-8d96-28798cfed535.png">

The thumbnail, title of the section ("KRAKEN THE CODE") and any hyperlink linked text in the description (e.g., MOUSE) might need some styling tweaks.